.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.12
+      - run: python -m pip install -r dev-requirements.txt
+      - run: python -m pip install ./fluent.syntax ./fluent.runtime
+      - run: python -m flake8
+      - run: python -m mypy fluent.syntax/fluent fluent.runtime/fluent
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.7, 3.8, 3.9, "3.10", 3.11, 3.12, pypy3.9, pypy3.10]
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - run: python -m pip install ./fluent.syntax ./fluent.runtime
+      - run: python -m unittest discover -s fluent.syntax
+      - run: python -m unittest discover -s fluent.runtime
+  baseline:
```

Comment:
Is there a more descriptive name for `baseline`. I see you're running a subset of unit tests, but it's somewhat unclear what is mean by baseline? If you are also running the same fluent.runtime unit tests in the "test" job, then what is the point of having "baseline"?

Is it something to do with the versions of fluent.syntax that are supported by python version 3.6?

Reply:
I would say `baseline_python` or `min_python_compatibility` would read more clearly based on what you said. This is a nit but while we are here right?